docs/related-projects/related-projects.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
I think you probably want to add this file to the sidebar list so that it shows up in the left-side TOC?
https://github.com/k3s-io/docs/blob/40e7e029624975cdbd3d5fca2dbb205bc6d0bbbb/sidebars.js#L1